src/test-run/index.js

```diff
@@ -247,5 +249,7 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
     if (this.lastDriverStatusResponse)
         return this.lastDriverStatusResponse;
 
-    return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
+    var responseTimeout = setTimeout(() => this._resolvePendingRequest(null), MAX_RESPONSE_DELAY);
```

Comment:
If we use `_resolvePendingRequest` wouldn't it break `lastDriverStatusResponse` machinery?